Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -384,4 +356,269 @@ static void _GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameSi
     }
 
     return lines;
+}
+
+/**
+ * Helper method to retrieve font fmaly names installed in the system.
+ *
+ * @return Unmutable array of font family name strings that are installed in the system.
+ */
+static NSArray<NSString*>* _DWriteGetFontFamilyNames() {
+    NSMutableArray<NSString*>* fontFamilyNames = [NSMutableArray<NSString*> array];
+
```

Comment:
Long-term we should probably cut out all NS stuff and convert everything to CF, but for now I'm doing whatever works best, as converting should be reasonably painless

---
In reply to: [78225402](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78225402) [](ancestors = 78225402,78221279,78220974)